test_wrappers.py

```
@@ -5317,7 +5318,18 @@ def test_go_yq(run):
             "foo_bar.yml",
             "table.json",
+            "data.yaml",
         ],
+        compare_results_with_expected={
+            "concat.yaml": "expected_concat.yaml",
+            "updated.yaml": "expected_updated.yaml",
+            "evaluated.yaml": "expected_evaluated.yaml",
+            "foo_bar.yml": "expected_foobar.yml",
+            "table.json": "expected_table.json",
+            "data.yaml": "expected_data.yaml",
+            "table.json": "expected_table.json",
```

Ruff (0.15.20)

[error] Dictionary key literal `"table.json"` repeated

Remove repeated key literal `"table.json"`

(F601)